docker/config/api/index.js

```diff
@@ -0,0 +1,175 @@
+const fs = require('fs');
+const winston = require('winston');
+
+const {
```

Comment:
I don't feel these redirections are necessary in this context (just reference `process.env.RABBITMQ_URL` directly). 

I also don't think we should be pulling these parameters from ENVs. I think we should have these values hardcoded in this configuration. Loading it from .env might be less bad, but then I don't think we need it in this context.

By the way, once we have our electron(?) configuration App, it should generate this file from user-provided info (bi directionally).  We won't need to do load parameters from ENV, nor wrap each component via if blocks.

Reply:
About redirection, it is about using it several times (so redir to keep clean), also helps knowing which variables will be accessed by the script.

About hard-coding in the config file and not using env: for docker-compose/development, it is good to keep all the configuration close together, especially when referring to a service name or port that was exposed to other containers. Loading from env also gives more flexibility for developers, since you don't need to bind files or change files to change the config, while you can export envvars, use `-e` in docker, envfiles, etc. In any case, this is only changes for docker-compose for development, and production/deployments can be kept hard-coded (can be even stored as a secret file and mounted to a path when using Kubernetes 😋).

The configuration app is more about running Brainlife locally for personal/lab usage, and not about development, is that right? The needs for these different personas seem different for me. I feel that for development, things would be better when more streamlines (and even turn-key, just clone'n'run you know?). 
Regarding if-blocks, I do not think both personas would like or would require to configure each app for authentication (Github, Google, ORCID, Globus, etc).

We were talking in Slack at some moment about having a mock-up public auth project with only user/pass for development, so we keep this on private and allow externals to play around with a simplified one. What you think? It may be the only exit for allowing developers without opening this repo, but I'm all ears.